documentation/basic/updates.md

```diff
@@ -1,9 +1,33 @@
 ---
 id: updates
 title: Updates
-description: "Data updates"
+description: 'Data updates'
 ---
 
+Everything you download with HACS will be create a dedicated [update entity](/docs/entities/update) for that repository.
+
+When a update is available for the repository, this will be shown in the Configuration panel of Home Assistant with other updates.

```

Comment:
For clarity, I would amend as below:

`When a update is available for the repository, it will be shown in the Configuration panel of the device.`

Reply:
Why?
This shows on the first page of the configuration panel (Now named "Settings"), there is no need to go to the device page for this.

Reply:
Oh you are totally right actually (with the new experimental branch). Maybe we should mention both ways.

Reply:
```suggestion
```

Removing as it is redundant, this is explained in <https://www.home-assistant.io/integrations/update/> which is linked from the next page.